### PR TITLE
Add generation of rpm for centos-7

### PR DESCRIPTION
**Pull request by axqx**
- Modified build.sh to generate rpm of medallia/aurora

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -14,52 +15,72 @@ echo "SNAPSHOT ${AURORA_SNAPSHOT}"
...
 BASE_DIR="$(pwd)"
+BUILD_PACKAGE=${BUILD_PACKAGE:-1}
 
-if [ ! -f aurora-scheduler_${AURORA_RELEASE}_amd64.deb ]; then
+function upload {
+  if [ ! -f $1 ]; then
+    echo "Cannot find package $1"
+    exit 1
+  fi
+  STATUS=""
```
Comment (mauri) [inline build.sh]: rm?

File: build.sh [hunk 2]
```diff
@@ -14,52 +15,72 @@ echo "SNAPSHOT ${AURORA_SNAPSHOT}"
...
+
   if [ ! -d aurora-packaging ]; then
       git clone "https://github.com/apache/aurora-packaging.git" "aurora-packaging"
   fi
 
   (cd aurora-packaging && \
-   git fetch origin "$AURORA_PACKAGE_BRANCH" && \
-   git checkout "$AURORA_PACKAGE_BRANCH" && \
-   ./build-artifact.sh builder/deb/ubuntu-trusty \
+  git fetch origin "$AURORA_PACKAGE_BRANCH" && \
```
Comment (mauri) [inline build.sh]: optional: the (cd ...) pattern is getting hard to read now, what about switching to `pushd aurora-packaging`, execute everything and then `popd`. We already set -e so any error would stop the script, so the `&&` are not really needed

**Comment by mauri** [comment]
LGTM!

File: build.sh (hunk 1 above)
Comment (axqx) [inline build.sh]: Fixed in 508b35ddefc8f042b07f3afdd0d44b7bb0753dbc

File: build.sh (hunk 2 above)
Comment (axqx) [inline build.sh]: Fixed in 508b35ddefc8f042b07f3afdd0d44b7bb0753dbc

**Pull request merged**